gecko.py

```diff
@@ -44,31 +45,25 @@ def fetch():
 
 
 def exchanges(dex):
-    data = [
-        {
-            "pair": ticker["coin_id"] + "<>" + ticker["target_coin_id"],
-            "volume": ticker["converted_volume"]["usd"],
-        }
-        for ticker in get("exchanges", dex)["tickers"]
-    ]
+    data = []
+    for page in range(1, 2):
+        tickers = get("exchanges", dex, "tickers", params={"page": page})["tickers"]
+        if not tickers:
+            break
+        data.extend(
+            {
+                "pair": ticker["coin_id"] + "<>" + ticker["target_coin_id"],
+                "volume": ticker["converted_volume"]["usd"],
+            }
+            for ticker in tickers
+            if not ticker["is_stale"]
+        )
     df = pd.DataFrame(data)
     df.set_index("pair", inplace=True)
     df.index.name = "pair"
     return df
 
 
-def exchanges_multi(dex, n_item=2):
-    df = exchanges(dex)
-    if n_item > 1:
-        for i in range(n_item):
-            tmp_df = exchanges(dex)
-            for j in tmp_df.index:
-                if j not in df.index:
-                    df.loc[j, "volume"] = tmp_df.loc[j, "volume"]
-
-    return df
-
```

Comment:
Why remove this function? It is meant for reduce the potential instability of data coming from cg endpoints.

Reply:
The point of this PR is that I have discovered the cause of the instability, and we can correct it by reading multiple pages.